server/lib/logging/route_logging.js

```diff
@@ -13,25 +13,45 @@ const config = require('../configuration');
  * Enhances connect logger middleware - custom formats.
  * See lib/configuration for usage.
  */
-const formats = {
-  'default_fxa': ':remote-addr - - ":method :url HTTP/:http-version" :status :response-time :res[content-length] ":referrer" ":user-agent"',
-
-  'dev_fxa': ':method :url :status :response-time'
-};
 
 // Used when logging is disabled
 const disabled = function (req, res, next) {
   next();
 };
 
+const formats = {
+  'default_fxa': (tokens, req, res) => JSON.stringify({
+    'content-length': tokens.res(req, res, 'content-length'),
+    HTTP: req.httpVersion,
+    method: tokens.method(req, res),
+    path: tokens.url(req, res),
+    referrer: req.headers['referer'],
+    'remote-addr': req.ip || req.connection.remoteAddress,
```

Comment:
I filed an issue to make this do a proper addressChain: https://github.com/mozilla/fxa-content-server/issues/5149

We can just keep `remoteAddressChain: req.ip || req.connection.remoteAddress,` in this PR for now and do a follow up.